### PR TITLE
board/stm32f0discovery: switched to OpenOCD

### PR DESCRIPTION
**Pull request by haukepetersen**
switched flashing and debug scripts to use OpenOCD

### CODE REVIEW

File: boards/stm32f0discovery/dist/debug.sh [hunk 1]
```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
 
-echo "Debugging $1"
-arm-none-eabi-gdb -tui -command=$1 $2
+openocd -f "board/stm32f0discovery.cfg" \
+    -c "tcl_port 6333" \
+    -c "telnet_port 4444" \
+    -c "init" \
```
Comment (thomaseichinger) [inline boards/stm32f0discovery/dist/debug.sh]: To keep things somewhat synchronised you could add something like [here](https://github.com/RIOT-OS/RIOT/pull/2364/files#diff-e0996e5f8c0e4adf96a4172c54084dc5R7)

File: boards/stm32f0discovery/Makefile.include [hunk 2]
```diff
@@ -37,9 +37,7 @@ export ASFLAGS += -ggdb -g3 $(CPU_USAGE) $(FPU_USAGE) -mlittle-endian
 export LINKFLAGS += -g3 -ggdb -std=gnu99 $(CPU_USAGE) $(FPU_USAGE) -mlittle-endian -static -lgcc -mthumb -mno-thumb-interwork -nostartfiles
 # $(LINKERSCRIPT) is specified in cpu/Makefile.include
 export LINKFLAGS += -T$(LINKERSCRIPT)
-export OFLAGS = -O binary
-export FFLAGS = write bin/$(BOARD)/$(APPLICATION).hex 0x08000000
-export DEBUGGER_FLAGS = $(RIOTBOARD)/$(BOARD)/dist/gdb.conf bin/$(BOARD)/$(APPLICATION).elf
+export OFLAGS = -O ihex
```
Comment (thomaseichinger) [inline boards/stm32f0discovery/Makefile.include]: Unrelated change.

**Comment by thomaseichinger** [comment]
Worked for me, two minor comments above. ACK anyway, merge at will.

File: boards/stm32f0discovery/Makefile.include (hunk 2 above)
Comment (haukepetersen) [inline boards/stm32f0discovery/Makefile.include]: nope, its related.

File: boards/stm32f0discovery/dist/debug.sh (hunk 1 above)
Comment (haukepetersen) [inline boards/stm32f0discovery/dist/debug.sh]: I don't really see the reason for this, normally you configure your gdb client (i.e. Eclipse) for a certain server port and not your server to the port your client wants...

Comment (gebart) [inline boards/stm32f0discovery/dist/debug.sh]: Sometimes you may want to run two boards in parallel, then it is convenient to be able to override the port assignment.

Comment (haukepetersen) [inline boards/stm32f0discovery/dist/debug.sh]: yap, you're right, That I haven't thought of. Will fix

**Comment by haukepetersen** [comment]
now the GDB port is configurable.

**Comment by haukepetersen** [comment]
@thomaseichinger, care to re-ACK?

**Comment by thomaseichinger** [comment]
re-ACKed, Travis doesn't test any of this functionality, go
